### DISCUSSION TITLE
USDZExporter from model-viewer

### DISCUSSION THREAD

**Discussion by danilokrulj** [google/model-viewer#4953] [Q&A]
I'm trying to allow for users to download a model they configured in the 3D view in multiple extensions. One of those extensions is supposed to be USDZ but I can't seem to set up threejs USDZExporter, but than I remembered model-viewer has USDZ conversion built in. 

Does anyone know where in the code is the USDZ conversion? I tried to find it but it seems I can't manage to do so. 

Also a version of threejs from which the USDZExporter is extracted would maybe help.

**Comment by elalish** [comment]
Try typing "USDZExporter" into Github's search box at the top of this screen. I believe the first result is what you're looking for.
